frontend/src/utils/scheduler/scheduler.ts

```diff
@@ -175,8 +175,8 @@ export const serializeSchedule = (schedule: Schedule, semester: Semester): Backe
  * Generates iCal file string.
  */
 export function scheduleToICal(schedule: Schedule, semester: Semester): string {
-	const SEMESTER_START = new Date(2023, 7, 23);
-	const LAST_COURSE_DAY = new Date(2023, 11, 1);
+	const SEMESTER_START = new Date(2024, 1, 16);
+	const LAST_COURSE_DAY = new Date(2024, 4, 26);
```

Comment:
Javascript `Date` month values go from 0 to 11 (see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date)